hack/local-up-kubeedge.sh

```diff
@@ -121,6 +123,7 @@ function start_edgecore {
   sed -i -e "s|token: .*|token: ${token}|g" \
       -e "s|hostnameOverride: .*|hostnameOverride: edge-node|g" \
       -e "s|/etc/|/tmp/etc/|g" \
+      -e "s|/var/lib/kubeedge/|/tmp&|g" \
```

Comment:
What's the meaning of `/tmp&`, for files under `/var/lib/kubeedge`, i'd prefer keep them here, not move to /tmp.

Reply:
> What's the meaning of `/tmp&`?

'&' means the whole matched string, in this context i.e. `/var/lib/kubeedge/`. 

>  For files under `/var/lib/kubeedge`, i'd prefer keep them here, not move to /tmp.
1. What's the difference between `/etc` and `/var/lib/kubeedge`? Why one is under `/tmp`, the other is not?
2. https://github.com/kubeedge/kubeedge/blob/29c26a61b3541331ae51917df83cbf6adcd7ea62/hack/local-up-kubeedge.sh#L48 needs to be changed if `/var/lib/kubeedge` keeps unchanged.

Reply:
Better to move all related files into a directory, also I can make it configurable if you wants.

Reply:
> &' means the whole matched string, in this context i.e. /var/lib/kubeedge/.

Get it, also fine to put under tmp.